src/components/ContentTemplate.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+	<NcAppContent :show-details="showDetails"
+		@update:showDetails="unselectItem()">
+		<template #list>
+			<NcAppContentList>
+				<div class="header">
+					<slot name="header" />
+				</div>
+
+				<FeedItemDisplayList :items="items"
+					:fetch-key="fetchKey"
+					:config="config"
+					@load-more="emit('load-more')" />
+			</NcAppContentList>
+		</template>
+
+		<NcAppContentDetails class="content-details-container">
+			<FeedItemDisplay v-if="selectedFeedItem" :item="selectedFeedItem" />
+		</NcAppContentDetails>
+	</NcAppContent>
+</template>
+
+<script setup lang="ts">
+
+import { PropType, computed, ref, watch } from 'vue'
```

Comment:
PropType doesn't exist, is this from Vue 3 maybe?

Reply:
No, it's exists but is a type definition, so probably has to be a type import.. I wonder why VS Code does not show me the eslint errors here.. 🤔